docs/src/api.md

```diff
@@ -52,19 +52,40 @@ nextid
 random_position
 ```
 
+### Removing agents
+```@docs
+kill_agent!
+genocide!
+sample!
+```
+
 ### Moving agents
 ```@docs
 move_agent!
 walk!
 ```
 
-### Removing agents
+## Path-finding
+In addition to the direct movement functions listed above, [`GridSpace`](@ref) has the additional benefit of path planning.
 ```@docs
-kill_agent!
-genocide!
-sample!
+AStar
+set_target!
+is_stationary
+walkmap
+heightmap
+find_path
 ```
```

Comment:
The location of `Path-finding` header in the documentation is out of place. Not only it interrupts the Model-agent interaction section, which is supposed to be followed by the discrete- or continous space exclusives, but is also really high up the sequence. I cannot imagine that path finding is more relevant than "Local area". Let's not forget that while this PR is large, and a big feature, it is not crucial for most agent based modelling scenarios.

Reply:
I think it should be the very last section in the API, after schedulers.

Reply:
Sure